apps/dashboard/src/locales/en/common/common.json

```diff
@@ -7,6 +7,8 @@
         "actions": "Actions",
         "gewisId": "GEWIS ID",
         "downloadPdf": "Download PDF",
+        "pdf": "PDF",
+        "till": "till",
```

Comment:
please remove the `till` key from other more specific translation modules if you want it in common.